### PR TITLE
LRQA-71119 | master

### PR DESCRIPTION
**Pull request by JingyuanZhao**
https://issues.liferay.com/browse/LRQA-71119
@JoyceWang08211

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/RemoteStagingWithVersioning.testcase [hunk 1]
```diff
@@ -90,6 +90,7 @@ definition {
 		Alert.viewInfoMessageSpecific(infoMessage = "There are no results.");
 	}
 
+	@description = "Able to publish Web Content via remote staging."
```
Comment (JoyceWang08211) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/RemoteStagingWithVersioning.testcase:93]: Case sensitivity issue

**Review by JoyceWang08211 — CHANGES_REQUESTED** [review]

**Comment by JingyuanZhao** [comment]
ci:test:sf

**Comment by JingyuanZhao** [comment]
ci:test:relevant

**Comment by JingyuanZhao** [comment]
ci:test:stable

**Comment by JingyuanZhao** [comment]
ci:test:staging

**Comment by JingyuanZhao** [comment]
ci:test:sf

**Comment by JoyceWang08211** [comment]
LGTM

**Comment by JoyceWang08211** [comment]
ci:forward

**Pull request closed without merging**